R/Lrnr_glm_semiparametric

```diff
@@ -0,0 +1,251 @@
+#' Semiparametric Generalized Linear Models
```

Comment:
Please adhere to the [80-column rule](https://softwareengineering.stackexchange.com/questions/148677/why-is-80-characters-the-standard-limit-for-code-width) throughout, via manual or automatic line-breaking (you can enable this in RStudio, if that's the IDE you're using). `styler::style_pkg()` also enforces this automatically iirc.